packages/coachmark/src/CoachIndicator.ts

```diff
@@ -60,25 +54,11 @@ export class CoachIndicator extends SpectrumElement {
             if (window.__swc.DEBUG) {
                 window.__swc.warn(
                     this,
-                    `The "variant" attribute/property of <${this.localName}> have been deprecated. Use "static" with any of the same values instead. "variant" will be removed in a future release.`,
+                    `The "variant" attribute/property of <${this.localName}> have been deprecated. Use "static-color" with any of the same values instead. "variant" will be removed in a future release.`,
```

Comment:
This deprecation notice should be removed?

Reply:
Same as below